codes/test_ppon.py

```diff
@@ -1,124 +0,0 @@
-import os
```

Comment:
Keep, will probably modify for a new image to image translation based upscale with WBC+PPON

Reply:
The code is fully identical to train.py (which uses Trainer), so if you plan to modify, just make your changes within train.py or the Trainer class, you can always expand the Trainer class or train.py to be able to specify which model type, e.g. `ppon` and do modifications for one specific model.

Reply:
Or you could even create a new class inheriting Trainer, and call it PPONTrainer.